src/integration/car/ev-database/EVDatabaseCarIntegration.ts

```diff
@@ -39,54 +40,96 @@ export default class EVDatabaseCarIntegration extends CarIntegration {
-            chargePhaseVolt: data.Charge_Standard_Table[chargeAlternative].Charge_PhaseVolt,
-            chargePhaseAmp: data.Charge_Standard_Table[chargeAlternative].Charge_PhaseAmp,
-            chargePhase: data.Charge_Standard_Table[chargeAlternative].Charge_Phase,
-            chargePower: data.Charge_Standard_Table[chargeAlternative].Charge_Power,
-            chargeTime: data.Charge_Standard_Table[chargeAlternative].Charge_Time,
-            chargeSpeed: data.Charge_Standard_Table[chargeAlternative].Charge_Speed,
+      if (chargeStandardTableUniversal) {
+        for (const chargeStandard of Object.keys(chargeStandardTableUniversal)) {
+          const chargeStandardTable: CarCatalogConverter = {
+            type: chargeStandard,
+            evsePhaseVolt: chargeStandardTableUniversal[chargeStandard].EVSE_PhaseVolt,
+            evsePhaseAmp: chargeStandardTableUniversal[chargeStandard].EVSE_PhaseAmp,
+            evsePhase: chargeStandardTableUniversal[chargeStandard].EVSE_Phase,
+            evsePhaseVoltCalculated: chargeStandardTableUniversal[chargeStandard].EVSE_Phase === 3 ? Voltage.VOLTAGE_400 : chargeStandardTableUniversal[chargeStandard].EVSE_PhaseVolt,
+            chargePhaseVolt: chargeStandardTableUniversal[chargeStandard].Charge_PhaseVolt,
+            chargePhaseAmp: chargeStandardTableUniversal[chargeStandard].Charge_PhaseAmp,
+            chargePhase: chargeStandardTableUniversal[chargeStandard].Charge_Phase,
+            chargePower: chargeStandardTableUniversal[chargeStandard].Charge_Power,
+            chargeTime: chargeStandardTableUniversal[chargeStandard].Charge_Time,
+            chargeSpeed: chargeStandardTableUniversal[chargeStandard].Charge_Speed,
           };
-          chargeAlternativeTables.push(chargeAlternativeTable);
+          chargeStandardTables.push(chargeStandardTable);
         }
-      }
-      if (data.Charge_Option_Table) {
-        for (const chargeOption of Object.keys(data.Charge_Option_Table)) {
-          const chargeAlternativeTable: CarCatalogConverter = {
-            type: chargeOption,
-            evsePhaseVolt: data.Charge_Standard_Table[chargeOption].EVSE_PhaseVolt,
-            evsePhaseAmp: data.Charge_Standard_Table[chargeOption].EVSE_PhaseAmp,
-            evsePhase: data.Charge_Standard_Table[chargeOption].EVSE_Phase,
-            chargePhaseVolt: data.Charge_Standard_Table[chargeOption].Charge_PhaseVolt,
-            chargePhaseAmp: data.Charge_Standard_Table[chargeOption].Charge_PhaseAmp,
-            chargePhase: data.Charge_Standard_Table[chargeOption].Charge_Phase,
-            chargePower: data.Charge_Standard_Table[chargeOption].Charge_Power,
-            chargeTime: data.Charge_Standard_Table[chargeOption].Charge_Time,
-            chargeSpeed: data.Charge_Standard_Table[chargeOption].Charge_Speed,
-          };
-          chargeOptionTables.push(chargeAlternativeTable);
+        if (data.Charge_Alternative_Table) {
```

Comment:
According to the spec this fiels no longer exists... So will always be empty.

Shouldn't we put all other Charge_Standard_Table_<COUNTRY> that wasn't used as standard (and make sure not null)

Reply:
Removed the chargeAlternativeTable since is no longer used.